### ISSUE TITLE
CLI flags not working

### ISSUE THREAD

**Issue by gschier** [aeirola/react-native-svg-app-icon#69]
Hi there! I'm using this within an Expo project so I don't want to have my config inside `app.json` (Expo uses different format). I tried using the CLI flags to configure instead, but it doesn't seem to work. It seems to ignore these and instead tries to read `./icon.svg`.

```sh
react-native-svg-app-icon --background-path=icon/icon-background.svg --foreground-path=icon/icon-foreground.svg
Running react-native-svg-app-icon
Reading file ./icon.svg
(node:50161) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, stat './icon.svg'
(Use `node --trace-warnings ...` to show where the warning was created)
(node:50161) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:50161) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by aeirola** [comment]
Hmm, interesting. That is definitely not the intended behaviour. I'll try to reproduce it when I find the time. Just to make sure, are you running the latest version 0.5.0?

**Comment by gschier** [comment]
Yes, that's the one. Thanks for the quick reply and awesome package 😃

**Comment by aeirola** [comment]
Okay, had time to take a look at it and definitely seems like the command line arguments never worked in the first place. Really sorry about this, I'll make a fix as soon as possible and release it.

**Comment by aeirola** [comment]
This has been fixed in 0.5.1 which was released just now. Sorry again for leaving the feature unfinished previously. And thanks for bringing the issue up. Now I've documented the existence of CLI flags and committed to keeping the functionality working.

For me at least, this works now.

```bash
% npx react-native-svg-app-icon --background-path=icon/icon-background.svg --foreground-path=icon/icon-foreground.svg

Running react-native-svg-app-icon
Reading file icon/icon-foreground.svg
```

**Comment by gschier** [comment]
Amazing, thanks for the quick turnaround! It's now working for me on 0.5.1 👍🏻
